client/cli/src/commands/run_cmd.rs

```diff
@@ -325,7 +325,7 @@ impl CliConfiguration for RunCmd {
 			Error::Input(format!(
 				"Invalid node name '{}'. Reason: {}. If unsure, use none.",
 				name, msg
-			));
+		))
```

Comment:
```suggestion
			))
```

@bkchr What was wrong? It corresponds to `Error::Input(format!(` :astonished: